src/views/Login/gordon-logo-vertical-white.svg

```diff
@@ -1,477 +1,477 @@
-		c0.466,0.439,0.76,1.062,0.76,1.763c0,0.343-0.07,0.676-0.205,0.975l0.008,0.006c-0.609,1.427-0.674,3.078-0.032,4.619
-		c0.636,1.545,1.844,2.668,3.28,3.242l-0.005,0.007c0.599,0.229,1.107,0.691,1.372,1.333c0.13,0.319,0.19,0.653,0.183,0.982h0.015
-		c-0.023,1.547,0.547,3.095,1.734,4.284c1.177,1.184,2.729,1.757,4.27,1.738v0.004c0.647-0.01,1.284,0.221,1.773,0.711
-		c0.246,0.239,0.434,0.531,0.547,0.835l0.007-0.004c0.573,1.439,1.703,2.65,3.248,3.297c1.544,0.634,3.198,0.571,4.621-0.036
-		l0.006,0.008c0.585-0.257,1.262-0.287,1.9-0.024c0.319,0.131,0.599,0.33,0.819,0.566l0.009-0.008
-		c1.082,1.114,2.589,1.799,4.261,1.799c1.669,0,3.186-0.685,4.26-1.801l0.003,0.002c0.441-0.458,1.055-0.741,1.743-0.741
-		c0.349,0,0.68,0.07,0.984,0.212l0.002-0.012c1.422,0.607,3.071,0.67,4.621,0.036c1.544-0.646,2.673-1.858,3.252-3.306
-		c0.233-0.589,0.692-1.085,1.322-1.344c0.326-0.138,0.663-0.194,0.994-0.179l0.005-0.015c1.542,0.019,3.092-0.554,4.271-1.738
-		c1.184-1.188,1.76-2.747,1.737-4.296h0.002c-0.008-0.634,0.227-1.269,0.706-1.75c0.255-0.25,0.55-0.435,0.856-0.556v-0.007
-		c1.434-0.571,2.639-1.697,3.272-3.237c0.642-1.549,0.578-3.21-0.036-4.632l0.008-0.004c-0.254-0.582-0.278-1.258-0.022-1.886
-		c0.142-0.335,0.345-0.615,0.586-0.844l-0.005-0.01c1.1-1.076,1.781-2.572,1.781-4.238c0-1.676-0.697-3.185-1.811-4.266l0.009-0.004
-		c-0.456-0.438-0.744-1.056-0.744-1.738c0-0.362,0.084-0.701,0.227-1.005l-0.005-0.009c0.591-1.416,0.648-3.055,0.013-4.593
-		c-0.638-1.542-1.854-2.667-3.297-3.241l0.006-0.011c-0.593-0.233-1.091-0.69-1.354-1.324c-0.146-0.334-0.197-0.676-0.186-1.021
-		h-0.006c0.006-1.536-0.568-3.071-1.737-4.243c-1.184-1.188-2.734-1.76-4.281-1.738V8.786c-0.64,0.013-1.279-0.215-1.77-0.704
-		c-0.253-0.25-0.439-0.552-0.558-0.869h-0.009c-0.58-1.424-1.7-2.62-3.231-3.255c-1.537-0.644-3.194-0.575-4.612,0.025l-0.006-0.008
-		c-0.585,0.26-1.27,0.292-1.904,0.027c-0.335-0.136-0.618-0.341-0.848-0.587L73.763,3.42c-1.077-1.095-2.569-1.775-4.228-1.775
-		c-1.672,0-3.168,0.687-4.25,1.792l-0.01-0.01c-0.436,0.468-1.059,0.761-1.753,0.761c-0.362,0-0.696-0.084-1.002-0.221l-0.005,0.013
-		c-1.418-0.597-3.06-0.659-4.596-0.025c-1.535,0.639-2.658,1.848-3.24,3.272l-0.007-0.002C54.444,7.831,53.98,8.34,53.334,8.6
-		c-0.327,0.139-0.669,0.198-1.002,0.186v0.013c-1.54-0.013-3.082,0.561-4.254,1.738c-1.182,1.172-1.759,2.722-1.738,4.256h-0.015
-		c0.021,0.645-0.215,1.292-0.705,1.786c-0.251,0.255-0.539,0.436-0.856,0.549l0.011,0.015c-1.437,0.572-2.644,1.702-3.28,3.234
-		c-0.639,1.538-0.582,3.186,0.024,4.603l-0.013,0.006c0.271,0.59,0.3,1.279,0.032,1.916c-0.13,0.331-0.332,0.603-0.567,0.827
-		l0.006,0.014c-1.104,1.077-1.793,2.586-1.793,4.25C39.185,33.661,39.87,35.158,40.972,36.24z M69.535,4.966
-		c14.845,0,26.883,12.035,26.883,26.88c0,14.848-12.038,26.883-26.883,26.883c-14.844,0-26.885-12.035-26.885-26.883
-		C42.65,17.001,54.691,4.966,69.535,4.966z"/>
-</g>
-</svg>
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Do you have any idea why this photo is showing as changed? It seems like nothing's actually different.

Reply:
We did not manually edit it or do anything that I know of, so I don't know where (or why) that "change" came from either